### ISSUE TITLE
Update the README

### ISSUE THREAD

**Issue by soumilk** [soumilk/Tkinter_and_Python_Game#3]
Update the [README] (https://github.com/soumilk/Python-GUI/blob/master/README.md) file of the repository. Add more resources and details about the python GUI workings.

**Comment by tushargoyal22** [comment]
what type of resources to be added?

**Comment by soumilk** [comment]
@b18030  Mainly learning resources, some good demo projects link might be helpful as well. Working of different components, usage, features

**Comment by Yugalbuddy** [comment]
Bro you can add turtle concept to make it more efficient.. After that add it to on hacktoberfest..

**Comment by soumilk** [comment]
yeah, thanks for suggestion @Yugalbuddy, I am just very occupied in other stuff but will definitely try to do it
